src/app/pages/index/components/list-requests/list-requests.component.ts

```diff
@@ -1,16 +1,78 @@
 import { Component, OnInit } from '@angular/core';
+import {DebugService} from '../../../../services/debug.service';
+import {HttpErrorResponse} from '@angular/common/http';
+import {MatIconRegistry} from '@angular/material/icon';
+import {DomSanitizer} from '@angular/platform-browser';
 
 @Component({
   selector: 'app-list-requests',
   templateUrl: './list-requests.component.html',
   styleUrls: ['./list-requests.component.css'],
 })
 export class ListRequestsComponent implements OnInit {
+  loading: boolean = false;
+  debugsList: any[] = [];
+  displayedColumns: string[] = ['position', 'tag', 'ip', 'method', 'isAjax', 'url', 'code', 'memory', 'time'];
 
-  constructor() {
+  constructor(private debugService: DebugService,
+              iconRegistry: MatIconRegistry,
+              sanitizer: DomSanitizer) {
+    iconRegistry.addSvgIcon(
+      'check',
+      sanitizer.bypassSecurityTrustResourceUrl('assets/img/yes-icon.svg'));
+    iconRegistry.addSvgIcon(
+      'ban',
+      sanitizer.bypassSecurityTrustResourceUrl('assets/img/no-icon.svg'));
   }
 
   ngOnInit(): void {
+    this.getDebugsList();
   }
 
+  getDebugsList() {
+    this.loading = true;
+    this.debugService.getList().subscribe(
+      response => {
+        this.loading = false;
+        if (response.success) {
+          this.debugsList = response.data;
+        } else {
+          console.log(response.error);
+        }
+      },
+      error => {
+        this.loading = false;
+        if (error instanceof HttpErrorResponse) {
+          console.log(error.statusText);
+        } else {
+          console.log(error);
+        }
+      }
+    );
+  }
+
+  formatMemory(memory: number) {
```

Comment:
set return types please
```suggestion
  formatMemory(memory: number): string {
```